apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -8,21 +8,39 @@ const examplePrompts = [
     id: 'random_mc',
     question:
```

Comment:
This `question` property is now unused, correct?

Reply:
Changed in 09ebd145316f730bdcbac5525887fda484b63483